README.md

```diff
@@ -1,17 +1,16 @@
 # gluster-kubernetes
 
-## Dynamic, persistent storage for Kubernetes with GlusterFS
+## GlusterFS + heketi on Kubernetes
```

Comment:
I disagree with this change of description.
The original description conveys what the project wants to achieve, the new description tells which components are used.